tests/test_edgeql_sql_codegen.py

```diff
@@ -37,7 +36,7 @@ class TestEdgeQLSQLCodegen(tb.BaseEdgeQLCompilerTest):
     """
 
     SCHEMA = os.path.join(os.path.dirname(__file__), 'schemas',
-                          'cards.esdl')
+                          'graphql.esdl')
```

Comment:
Could you switch this back to `cards`, restore the original `no_self_join` test, and then add your modified test back in as a new test, except using the `issues` schema instead of `graphql` (using `Issue.priority`, maybe?)

I like adding a test that uses a `single` link, but would like to keep the `multi` link test also.

And `issues` and `cards` are the most widely used test schemas internally, so I'd rather just use those when possible, since they are more immediately familiar.

See `test_codegen_elide_optional_wrapper` below for an example of accessing the issues schema; the names to pass to `schema.get` will be `issues::Issue` (instead of `default::`).

Reply:
Oh, it could also use `cards` and `User.avatar`

Reply:
Oh, or we could modify this test suite to just *always* use `issues` and drop the use of `cards` (replacing `User.deck.name` with `User.todo.name`, maybe).

That's probably the right thing to do, but you don't need to do it as part of this PR.

(I usually reach for `cards` as my go to database when testing things, because I like the theming and because I think the test *data* for it is better, but `issues` is probably a better test schema in general because it exercises the feature set better; `cards` doesn't have any optional properties, which is why I pulled in `issues` also, but probably I should have just switched to `issues` when I did that)